query/src/main/scala/filodb/query/LogicalPlan.scala

```diff
@@ -274,9 +274,9 @@ final case class ScalarVaryingDoublePlan(vectors: PeriodicSeriesPlan,
   */
 final case class ScalarTimeBasedPlan(function: ScalarFunctionId, rangeParams: RangeParams) extends ScalarPlan {
   override def isRoutable: Boolean = false
-  override def startMs: Long = rangeParams.start
-  override def stepMs: Long = rangeParams.step
-  override def endMs: Long = rangeParams.end
+  override def startMs: Long = rangeParams.start * 1000
+  override def stepMs: Long = rangeParams.step * 1000
+  override def endMs: Long = rangeParams.end * 1000
```

Comment:
Thanks for the fix! Can we rename rangeParams fields to `startSecs`, `endSecs` etc ?

Reply:
ack, fixed it